nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -307,20 +304,20 @@
 - module-name: "clr"
   dlls:
     - dest_path: "."
-      dir: "."
-      include_from_filenames: ""
-      patterns:
-        - "Python.Runtime"
+      from_filenames:
+        "dir": "."
```

Comment:
Should this not be `relative_path` if anything, and for `.` it also should be removed, as that's the default.